--- C:/PROJECTS/itext.orig/src/core/com/itextpdf/text/pdf/PdfChunk.java	Sun Dec  6 16:39:56 2009
+++ C:/PROJECTS/itext/src/core/com/itextpdf/text/pdf/PdfChunk.java	Sun Jan 24 18:06:50 2010
@@ -276,7 +276,8 @@
     protected int getWord(String text, int start) {
         int len = text.length();
         while (start < len) {
-            if (!Character.isLetter(text.charAt(start)))
+            //if (!Character.isLetter(text.charAt(start)))
+            if (Character.isSpace(text.charAt(start)))
                 break;
             ++start;
         }
