--- C:/PROJECTS/itext.orig/src/core/com/itextpdf/text/pdf/PdfDocument.java	Sun Dec  6 16:39:56 2009
+++ C:/PROJECTS/itext/src/core/com/itextpdf/text/pdf/PdfDocument.java	Mon Dec 21 22:01:25 2009
@@ -1238,7 +1238,7 @@
     }
 
     /** The characters to be applied the hanging punctuation. */
-    static final String hangingPunctuation = ".,;:'";
+    static final String hangingPunctuation = ".,;:'-";
 
     /**
      * Writes a text line to the document. It takes care of all the attributes.
@@ -2331,7 +2331,8 @@
         boolean underlying = (image.getAlignment() & Image.UNDERLYING) == Image.UNDERLYING;
         float diff = leading / 2;
         if (textwrap) {
-            diff += leading;
+            //diff += leading;
+            diff = image.getSpacingBefore();
         }
         float lowerleft = indentTop() - currentHeight - image.getScaledHeight() -diff;
         float mt[] = image.matrix();
